neqo-bin/src/bin/server/main.rs

```diff
@@ -260,14 +261,7 @@ struct SimpleServer {
 }
 
 impl SimpleServer {
-    const MESSAGE: &'static [u8] = b"I am the very model of a modern Major-General,\n\
-        I've information vegetable, animal, and mineral,\n\
-        I know the kings of England, and I quote the fights historical\n\
-        From Marathon to Waterloo, in order categorical;\n\
-        I'm very well acquainted, too, with matters mathematical,\n\
-        I understand equations, both the simple and quadratical,\n\
-        About binomial theorem, I'm teeming with a lot o' news,\n\
-        With many cheerful facts about the square of the hypotenuse.\n";
+    const MESSAGE: &'static [u8] = &[0; 4096];
```

Comment:
To the Gilbert and Sullivan fans: should I try to find a larger excerpt from the opera, rather than a boring `[0; 4096]`?

https://en.wikipedia.org/wiki/Major-General%27s_Song

Reply:
`git blame` says this came from @martinthomson. I don't care :-)

Reply:
If anyone cares, they can do a PR to propose some new text.